test/units/TEST-13-NSPAWN.machined.sh

```diff
@@ -399,7 +399,7 @@ grep -q "BAR" /tmp/none-existent-file
 kill "$PID"
 
 # Test varlinkctl's --exec fd passing logic properly
-assert_eq "$(varlinkctl --exec call /run/systemd/machine/io.systemd.Machine io.systemd.Machine.Open '{"name": ".host", "mode": "shell", "user": "root", "path": "/bin/bash", "args": ["/bin/bash", "-c", "echo $((7 + 8))"]}' -- bash -c 'read -r -N 2 x <&3 ; echo "$x"')" 15
+assert_eq "$(varlinkctl --exec call /run/systemd/machine/io.systemd.Machine io.systemd.Machine.Open '{"name": ".host", "mode": "shell", "user": "root", "path": "/bin/bash", "args": ["/bin/bash", "-c", "echo $((7 + 8))"], "environment": ["TERM=dumb"]}' -- bash -c 'read -r -N 2 x <&3 ; echo "$x"')" 15
```

Comment:
The change looks in a wrong commit ??

Reply:
this is the right commit actually. the Open() call with a mode of "shell" asks for a pty to be allocated, i.e. the full interactive stuff with ANSI sequences and stuff. Except that we actually want to do byte precise readings, without ANSI sequences and hence have to turn them off via TERM=dumb explicitly.

i'll add an explanation to the commit msg, why this belongs here: after all this commit adds the new OSC sequences to ptys associated with services.